.github/workflows/latest-release.yml

```diff
@@ -37,6 +37,22 @@ jobs:
 
           ./contribution/k3s/install_k3s.sh
 
+      - name: Setup local registry
```

Comment:
let's directly use YAML files in [annotations](https://github.com/kubearmor/KubeArmor/tree/main/deployments/annotations).